jobs/job_options.py

```diff
@@ -0,0 +1,62 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Option class for configuring the behavior of Oppia jobs."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import feconf
+
+import apache_beam as beam
+from apache_beam.options import pipeline_options
+
+
+class JobOptions(pipeline_options.GoogleCloudOptions):
+    """Option class for configuring the behavior of Oppia jobs."""
+
+    def __init__(
+            self,
+            project=feconf.OPPIA_PROJECT_ID,
+            region=feconf.GOOGLE_APP_ENGINE_REGION,
+            **kwargs):
```

Comment:
Would it be possible to enumerate allowed options (with documentation) rather than allow literally anything to be passed in?

Reply:
Can we throw an error if args that don't meet that criterion are changed?

Reply:
Done.